style.css/sixthsectionStyle.css

```diff
@@ -0,0 +1,27 @@
+.sixth-section {
+  text-align: center;
```

Comment:
This section needs gray background. Check the design.

Reply:
Oh!I changed it.